manageiq-providers-openstack.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "activesupport",        ">= 5.0", "< 5.3"
+  s.add_runtime_dependency "activesupport",        "~> 5.2.4", ">= 5.2.4.3"
```

Comment:
@djberg96 try `"activesupport", ">= 5.2.4.3", "~> 5.2.4"`  I've seen hakiri not process all the versions passed the same way that bundler does and helps to have the unaffected version first.